docs/content/en/docs/workshops/packages/adot/adot_amp_amg.md

```diff
@@ -0,0 +1,612 @@
+                    - action: replace
+                      regex: (https?)
+                      source_labels:
+                      - __meta_kubernetes_pod_annotation_prometheus_io_scheme
+                      target_label: __scheme__
+                    - action: replace
+                      regex: (.+)
+                      source_labels:
+                      - __meta_kubernetes_pod_annotation_prometheus_io_path
+                      target_label: __metrics_path__
+                    - action: replace
+                      regex: ([^:]+)(?::\d+)?;(\d+)
+                      replacement: $$1:$$2
+                      source_labels:
+                      - __address__
+                      - __meta_kubernetes_pod_annotation_prometheus_io_port
+                      target_label: __address__
+                    - action: labelmap
+                      regex: __meta_kubernetes_pod_annotation_prometheus_io_param_(.+)
+                      replacement: __param_$1
+                    - action: labelmap
+                      regex: __meta_kubernetes_pod_label_(.+)
+                    - action: replace
+                      source_labels:
+                      - __meta_kubernetes_namespace
+                      target_label: namespace
+                    - action: replace
+                      source_labels:
+                      - __meta_kubernetes_pod_name
+                      target_label: pod
+                    - action: drop
+                      regex: Pending|Succeeded|Failed|Completed
+                      source_labels:
+                      - __meta_kubernetes_pod_phase
+
+            processors:
+              batch/metrics:
+                timeout: 60s
+
+            exporters:
+              logging:
+                logLevel: info
+              prometheusremotewrite:
+                endpoint: "<AMP-WORKSPACE>/api/v1/remote_write" # Replace with your AMP workspace
+                auth:
+                  authenticator: sigv4auth
+            
+            service:
+              extensions:
+                - health_check
+                - memory_ballast
+                - sigv4auth
+              pipelines:
+                metrics:
+                  receivers: [prometheus]
+                  processors: [batch/metrics]
+                  exporters: [logging, prometheusremotewrite]
+
+      ```
+    </details>
+
+1. Bind additional roles to the service account `pod-identity-webhook` (created at step [Create an EKS-A cluster with IRSA](#create-an-eks-a-cluster-with-irsa)) by applying the following file in the cluster (using `kubectl apply -f <file-name>`). This is because `pod-identity-webhook` by design does not have sufficient permissions to scrap all Kubernetes targets listed in the ADOT config file above. If modifications are made to the Prometheus Receiver, make updates to the file below to add / remove additional permissions before applying the file.
```

Comment:
```suggestion
1. Bind additional roles to the service account `pod-identity-webhook` (created at step [Create an EKS-A cluster with IRSA](#create-an-eks-a-cluster-with-irsa)) by applying the following file in the cluster (using `kubectl apply -f <file-name>`). This is because `pod-identity-webhook` by design does not have sufficient permissions to scrape all Kubernetes targets listed in the ADOT config file above. If modifications are made to the Prometheus Receiver, make updates to the file below to add / remove additional permissions before applying the file.
```

Reply:
Updated